layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/FontAwesomePickerDrawer.vue

```diff
@@ -0,0 +1,196 @@
+        </v-icon>
+        <span class="ms-2"> {{ $t('siteNavigation.label.selectIcon.title') }}</span>
+      </div>
+    </template>
+    <template slot="content">
+      <v-card-text class="d-flex pb-2">
+        <v-text-field
+          v-model="keyword"
+          placeholder="filter by name"
+          prepend-inner-icon="fa-filter"
+          class="inputPeopleFilter"
+          @keyup="filterIcons($event)" />
+      </v-card-text>
+      <v-card-text class="d-flex pb-2">
+        <v-container id="iconPicker">
+          <v-row class="ma-0">
+            <v-col 
+              v-for="icon in icons"
+              :key="icon.iconValue"
+              cols="3"
+              class="pa-0 pa-1">
+              <v-sheet 
+                class="rounded-lg clickable "
+                :class="selectedIcon === icon ? 'grey-background' : 'light-grey-background'"
+                @click="selectIcon(icon)">
+                <div class="d-flex flex-grow-1">
+                  <v-icon
+                    v-if="selectedIcon === icon"
+                    color="primary"
+                    size="18"
+                    class="pb-8 py-1 pl-1">
+                    fas fa-check-square
+                  </v-icon>
+                  <v-icon
+                    size="32"
+                    class="flex-grow-1 align-center pt-6 pb-2"
+                    color="black">
+                    fas fa-{{ icon.iconValue }}
+                  </v-icon>
+                </div>
+                <v-tooltip bottom>
+                  <template #activator="{ on, attrs }">
+                    <p                 
+                      v-on="on"
+                      v-bind="attrs"
+                      class="align-center text-truncate text-caption text--primary font-weight-medium pb-2 mx-1">
+                      {{ icon.iconName }}
+                    </p>
+                  </template>
+                  <span>{{ icon.iconName }}</span>
+                </v-tooltip>
+              </v-sheet>
+            </v-col>
+          </v-row>
+        </v-container>
+      </v-card-text>
+      <v-card-text class="align-center">
+        <v-btn
+          v-if="!search"
+          elevation="0"
+          @click="showMoreIcons">
+          {{ $t('siteNavigation.label.btn.showMore') }}
+        </v-btn>
+      </v-card-text>
+    </template>
+    <template slot="footer">
+      <div class="d-flex justify-end">
+        <v-btn
+          class="btn ms-2"
+          @click="close">
+          {{ $t('siteNavigation.label.btn.cancel') }}
+        </v-btn>
+        <v-btn
+          :loading="loading"
+          :disabled="disabled"
+          class="btn btn-primary ms-2"
+          @click="saveIcon">
+          {{ $t('siteNavigation.label.btn.save') }}
+        </v-btn>
+      </div>
+    </template>
+  </exo-drawer>
+</template>
+
+
+<script>
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fas } from '@fortawesome/free-solid-svg-icons';
```

Comment:
no imports allowed directly in vuejs files:
https://github.com/Meeds-io/meeds/blob/develop/CODING_GUIDELINES.md#devstd-29